Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -1789,15 +1789,15 @@ - (void)_storeAppHighlight:(const WebCore::AppHighlight&)highlight
 #endif
 
 #if ENABLE(UNIFIED_TEXT_REPLACEMENT)
-- (void)_addTextIndicatorStyleForID:(NSUUID *)nsUUID withStyleType:(WKTextIndicatorStyleType)styleType
+- (void)_addTextIndicatorStyleForID:(NSUUID *)nsUUID withData:(const WebKit::TextIndicatorStyleData&)data
 {
 #if PLATFORM(IOS_FAMILY)
-    [_contentView addTextIndicatorStyleForID:nsUUID withStyleType:styleType];
+    [_contentView addTextIndicatorStyleForID:nsUUID withStyleType:(WKTextIndicatorStyleType)data.style];
```

Comment:
I don't think this cast is that safe since they are different types

Reply:
Right, is there a reason why we need both enums? (maybe it was for some Swift-interop?)

But if we do need both, we should have a method to convert between them, similar to `static inline OptionSet<WebKit::FindOptions> toFindOptions(_WKFindOptions wkFindOptions)`.

Reply:
Yes, we need both, I will make a converter.